data_structures/linked_list/linked_list.py

```diff
@@ -0,0 +1,58 @@
+class Node:
+    
+    def __init__(self, data=None):
+        self.data = data
+        self.next = None
+
+class LinkedList:
+
+    def __init__(self):
+        self.front = None
+        self.rear = None
+        self.size = 0        
+
+    def insert(self, data):
```

Comment:
The insertion here should provide an `index` that allows inserting `data` at arbitrary position.